File: app/src/main/java/com/example/aleksefo/deardiary/app/RealmApplication.java
package com.example.aleksefo.deardiary.app;

import android.app.Application;
import io.realm.Realm;
import io.realm.RealmConfiguration;

public class RealmApplication extends Application {

	@Override
	public void onCreate() {
		super.onCreate();
		Realm.init(this);
		RealmConfiguration realmConfiguration = new RealmConfiguration.Builder().build();
		Realm.setDefaultConfiguration(realmConfiguration);
//		Realm.deleteRealm(realmConfiguration);
	}
}

File: app/src/main/java/com/example/aleksefo/deardiary/app/RealmApplication.kt
package com.example.aleksefo.deardiary.app

import android.app.Application
import io.realm.Realm
import io.realm.RealmConfiguration

class RealmApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Realm.init(this)
        val realmConfiguration = RealmConfiguration.Builder().build()
        Realm.setDefaultConfiguration(realmConfiguration)
        //		Realm.deleteRealm(realmConfiguration)
    }
}
